Sources/TuistGenerator/Generator/BuildPhaseGenerator.swift

```diff
@@ -152,7 +152,8 @@ final class BuildPhaseGenerator: BuildPhaseGenerating {
                 shellPath: script.shellPath,
                 shellScript: script.shellScript(sourceRootPath: sourceRootPath),
                 runOnlyForDeploymentPostprocessing: script.runForInstallBuildsOnly,
-                showEnvVarsInLog: script.showEnvVarsInLog
+                showEnvVarsInLog: script.showEnvVarsInLog,
+                dependencyFile: script.dependencyFile?.pathString
```

Comment:
```suggestion
                dependencyFile: script.dependencyFile.map { $0.relative(to: sourceRootPath).pathString }
```

Reply:
Should I update one of the tests to also check for the dependency file?